docs/Inside_Arbitrum_Nitro/Inside_Arbitrum_Nitro.md

```diff
@@ -0,0 +1,527 @@
+---
+id: Inside_Arbitrum_Nitro
+title: Inside Arbitrum Nitro
+sidebar_label: Inside Arbitrum Nitro
+---
+
+This document is a deep-dive explanation of Arbitrum Nitro’s design and the rationale for it. This isn’t API documentation, nor is it a guided tour of the code--look elsewhere for those. “Inside Arbitrum Nitro” is for people who want to understand Nitro's design.
+
```

Comment:
Maybe add a catch-all caveat up front r.e. rollup vs anytrust, e.g.,:


> This document will be focused on the Arbitrum Nitro Rollup protocol; the Arbitrum Nitro AnyTrust protocol works very similar and uses much of the same code, but relies on a different trust assumption for data availability; for more on AnyTrust, see [Inside AnyTrust](inside_anytrust).

Reply:
Integrated the "Inside AnyTrust" content at the end of this document, and clarified Rollup vs AnyTrust in the intro.